students/danwoj/Session11/test_generators.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+from generators import soi, doubler, fib, primes, ecount
+
+def test_soi():
+    '''
+    Tests the Sum of Integers function
+    '''
+    start_value = 0
+    stop_value = 10
+    x = soi(start_value, stop_value)
+    y = next(x)
+    for i in range(stop_value-1):
```

Comment:
you can probably simply do:

for i in x:

here -- x is an iterator, so it should pick up where you left off in the for loop.

As long as it will terminate! Which in this case it does.